orc8r/cloud/go/services/ctraced/obsidian/models/swagger.v1.yml

```diff
@@ -177,6 +177,7 @@ definitions:
         x-nullable: false
         enum:
           - 'GATEWAY'
+          - 'GATEWAY_CUSTOM'
```

Comment:
minor nit: custom option isn't a separate trace type. Doesn't seem right to add GATEWAY_CUSTOM as a separate trace type